java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -240,6 +245,29 @@ export function activate(context: ExtensionContext) {
 
 }
 
+function showStatusBarMessage(params : ShowStatusMessageParams) {
+    let decorated : string = params.message;
+    let timeout : number = 10000;
```

Comment:
Shoudn't the timeout be read from `params.timeout`, if present?

Reply:
victim of the code cleanup :(